src/App.js

```diff
@@ -708,6 +712,16 @@ class App extends Component {
                   <Bx2PanelValidatorsThemed nodes={this.state.nodes} />
                 )}
               />
+              <Route
```

Comment:
validator detail page takes list of nodes already in state & specific id from path